rpc/common/moc_common_nodeinfo.proto

```diff
@@ -27,18 +31,23 @@ message GPU {
 	uint32 count = 3;
 }
 
+message OperatingSystemInfo {
+	uint64 operatingsystemsku = 1;
+}
+
 message NodeInfo {
 	string name = 1;
 	string id = 2;
 	Resources capability = 3;
 	Resources availability = 4;
-	OperatingSystemType ostype = 6;
-	Status status = 7;
+	OperatingSystemType ostype = 5;
```

Comment:
Same here. Can you keep the previous values to avoid introducing a breaking change?

Reply:
Yes, I updated it to previous values.